Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -147,6 +147,17 @@ def setName(self, string, nameID, platformID, platEncID, langID):
 		else:
 			self.names.append(makeName(string, nameID, platformID, platEncID, langID))
 
+	def removeNames(self, nameID=None, platformID=None, platEncID=None, langID=None):
+		"""Remove name records identified by 'nameID', 'platformID', 'platEncID'
+		and 'langID'.
+		"""
+		self.names = [n for n in self.names
```

Comment:
I prefer if add a newline after the opening `[`, with a single indentation level